less/home-page.less

```diff
@@ -71,5 +89,24 @@
         border-radius: 10px;
       }
     }
+    @media @phone{
+      display: flex;
+      flex-flow: column-reverse nowrap;
+      justify-content: center;
+      padding: 5px;
+      margin-left: 1%;
+      .walk-text{
+        width: 98%;
+        margin: 0;
+        .btn {
+          .btn-class(100%, 60px);
+        }
+      }
+      .walk-img{
```

Comment:
could have just set the order property on the image to -1; But good use of flex box reverse column.